docs/README.md

```diff
@@ -1,9 +1,20 @@
-# btg360
+# BTG360
 
-VTEX BTG360 first-party app
+BTG 360 uses behavioral targert technology so you can achieve even more assertive results
+with email marketing campaigns based on browsing and / or purchasing behavior of each
+one of the registered users in your lists.
+For more information please visit, [AllIn](https://allinmail.com.br/) website.
 
 ## How to install
 
 Open the VTEX App Store and install this app on your store.
 
 Now, open the app settings on your admin and fill in the BTG360 options.
+
+or
+
+Run the following command:
+
+```sh
+vtex install vtex.rtbhobtg360use@0.x
+```
```

Comment:
You don't need the root readme file

Reply:
there's no readme file in the root of btg360.

Reply:
Sorry, my mistake, misread the file path